_data/changelogs/component-header.yml

```diff
@@ -2,6 +2,16 @@ title: Header
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Reordered header items so that search bar appears last in mobile header.
+    summaryAdditional: Now, the header is in compliance with WCAG criterion 2.4.3 (Focus order) in mobile view. 
```

Comment:
**suggestion:** Wanted to offer an alternate suggestion for the `summaryAdditional` element. I think this is a bit more explicit and also gives next steps. 

```suggestion
    summary: Reordered header items so that search bar appears last in mobile header. 
    summaryAdditional: Now, the visual order of the component matches the tab order. If you would like to visually keep the search bar at the top of the menu, you will need to reorder your markup in the mobile view.
```